tests/integration/session/test_login.py

```diff
@@ -91,3 +112,27 @@ def test_login_with_missing_mandatory_claims_should_be_forbidden(self):
 
         # Then
         self.assertStatusForbidden()
+
+    def test_login_token_with_survey_url_should_redirect_to_survey(self):
+
+        survey_url = "http://eq-survey-register/my-test-schema"
+
+        # Given
+        token = self.token_generator.create_token_with_survey_url('0205', '1', survey_url)
```

Comment:
Should we start moving away from using "real" surveys for testing? i.e. any new tests/modified tests are switched to use a suitable test_* fixture as we do other work.